consent-ui/src/app/ais/sca-result-page/sca-result-page.component.spec.ts

```diff
@@ -30,10 +35,26 @@ describe('ReportScaResultComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(ReportScaResultComponent);
     component = fixture.componentInstance;
+    consentAuthorizationService = fixture.debugElement.injector.get(ConsentAuthorizationService);
     fixture.detectChanges();
+    form = component.reportScaResultForm;
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should be true if the reportScaResultForm is invalid', () => {
+    consentAuthorizationServiceSpy = spyOn(consentAuthorizationService, 'embeddedUsingPOST').and.returnValue(of());
+    form.get('tan').setValue('');
+    component.submit();
+    expect(component.reportScaResultForm.invalid).toBe(true);
+  });
+
+  it('should be true if the reportScaResultForm is valid', () => {
+    consentAuthorizationServiceSpy = spyOn(consentAuthorizationService, 'embeddedUsingPOST').and.returnValue(of());
+    form.get('tan').setValue(StubUtilTests.SCA_METHOD_VALUE);
+    component.submit();
+    expect(component.reportScaResultForm.valid).toBe(true);
```

Comment:
It is good to add expectation that we call backend on submit and (optionally) that we move somewhere based on its response